packages/nocodb/src/lib/db/sql-data-mapper/lib/sql/BaseModelSqlv2.ts

```diff
@@ -70,9 +70,9 @@ class BaseModelSqlv2 {
   private _columns = {};
 
   private config: any = {
-    limitDefault: 25,
-    limitMin: 1,
-    limitMax: 1000
+    limitDefault: process.env.DB_QUERY_LIMIT_DEFAULT || 25,
```

Comment:
We need to handle negative and invalid numbers here, something like the following we can do.

```js
limitDefault: Math.max(+process.env.DB_QUERY_LIMIT_DEFAULT || 25,1)
```